hack/gencerts.sh

```diff
@@ -17,6 +17,61 @@
 # Generates a CA certificate, a server key, and a server certificate signed by the CA.
 
 set -e
+SCRIPT=`basename ${BASH_SOURCE[0]}`
+
+function usage {
+  cat<< EOF
+  Usage: $SCRIPT
+  Options:
+  -h | --help               	   Display help information.
+  -n | --namespace <namespace>   The namespace where the Spark operator is installed.			
+  -p | --pod-container           whether the script is running inside a pod container or manually invoked.
```

Comment:
Sorry one last comment, I think it's better to rename this to `--in-pod`, and change the description to `Whether the script is running inside a pod or not.`.

Reply:
Sure. Fixed.